### DISCUSSION TITLE
Welcome to BBLL Discussions!

### DISCUSSION THREAD

**Discussion by xiaye13579** [xiaye13579/BBLL#131] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
过年可不可以对bbll进行一次UI大更新大改版呢

**Comment by 1234fghj** [comment]
能不能添加其他应用的解码，有些视频会黑屏会出现source error

**Comment by llhdxnq** [comment]
以后会加入评论区功能吗？

**Comment by loveyou-3000** [comment]
不知道啥原因，一旦播放竖屏类型的视频就会大概率白屏或者绿屏。然后正常再播放其它视频就都是白屏或者绿屏，不过声音从头到尾都正常。更换分辨率、编码方式、退出应用后台都不能解决，只有重启电视。至少试过的1.2.6和1.2.7都有出现，电视是OPPO k9 不知道是否是个例。

**Comment by L1uuu** [comment]
可以将这款软件推广给更多人吗？比如在B站发布视频，B站好想对非盈利第三方客户端还是持开放态度的，比如wiliwili作者也在B站发视频宣传。这么好的应用值得被更多人知道

**Reply by ghost** [reply]
你想多了，现在必须以低调实用为主，不得大量宣传，如果有这么多人使用的话，那大佬的服务器还能受得了吗

**Reply by ghost** [reply]
你想想如果用的人越来越多服务器会不会卡

**Reply by ghost** [reply]
所以不要乱宣传

**Reply by L1uuu** [reply]
@xiaye13579 这个是作者的意思么？或什么时候表达过这个意思么？想听听作者的态度

**Reply by ghost** [reply]
我说的不是作者表达的问题是服务器的容量问题知道吧

**Reply by ghost** [reply]
所以现在必须以低调使用为主
等作者做得更好之后功能更全面了之后
咱们再把这个软件推到b站
行不 @xiaye13579

**Reply by Apricityx** [reply]
低调使用确实是对的，但原因并不是服务器负载的问题

**Comment by rockenergy** [comment]
**_希望出个    西瓜视频tv     版本_**

**Comment by long1512448773** [comment]
1:可以增加一个账号切换功能 因为是TV端 我妹也想看
2:优化一下视频 有些视频黑屏无法播
3:可以增加观看UP主粉丝功能吗？
4:能不能 多一些解码！因为有些缓存比较慢，不是网络问题
5:关注的人直播显示不多啊！直播清晰度有待加强 看高清的画质，还不如看原画质一言难尽啊！ 能不能默认108p呢起码看清👍👍👍👍👍❤️❤️❤️❤️

**Comment by ghost** [comment]
建议作者不要开发西瓜视频，因为西瓜视频也没啥好看的












达瓦里氏
***@***.***


---Original---
From: ***@***.***&gt;
Date: Sun, Feb 19, 2023 15:27 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [xiaye13579/BBLL] Welcome to BBLL Discussions! (Discussion #131)




 
希望出个    西瓜视频tv     版本

**Comment by gudianxingkongmeng** [comment]
会做竖屏版吗

**Comment by yilai1245** [comment]
建议搞个后台播放声音，很需要这个吗

**Comment by yangqiaoqiang** [comment]
问一下，这种不会被叔叔告侵权吗。

**Comment by jadexb** [comment]
有两个建议：
1、关于支持的版本。release支持5.0以上是不是太保守了？会不会换成9.0之类的甚至全64位版本，性能会更加好一点？
2、建议搞轻巧小快，重点就是要性能好，太花里胡哨的功能不要加太多，重点做好性能和没有BUG。
希望能用上更加好的作品。感谢。

**Comment by Tonglifornia** [comment]
各位 谁能拉我进qq群.....

**Comment by aihuijin** [comment]
首先感谢作者开发了这么好的软件，现在有个需求，看看是否可以加到下一个版本里。
首页显示动态，播放完成后播放列表下一个，这都通过设置完成了，就是没有自动播放。
就是点开app，就自动播放动态列表，可以吗？

**Comment by linpyang** [comment]
请问什么时候可以支持管理历史记录

**Comment by linpyang** [comment]
还有一个问题，有的分段合集只可以显示第一段（如黄夫人的高一高二合集）

**Comment by tilby** [comment]
不知道大家有没有遇到一个问题，就是经常会播放下一P时卡死，timeout，然后它会再跳到再下一P尝试；
如果退出去重新进来，手动选刚刚卡死的那一P，经常是可以的。
同一个视频我用手机APP没有问题，甚至用其它第三方APP也没有这个问题；

**Comment by qwr5650** [comment]
什么时候更新BBLL新版本？@xiaye13579
